clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/TransportResourceDefinition.java

```diff
@@ -241,7 +243,50 @@ public PathAddress transform(PathAddress address) {
     @Override
     public void register(ManagementResourceRegistration parentRegistration) {
         ManagementResourceRegistration registration = parentRegistration.registerSubModel(this);
-        parentRegistration.registerAlias(LEGACY_PATH, new SimpleAliasEntry(registration));
+        parentRegistration.registerAlias(LEGACY_PATH, new AliasEntry(registration) {
+            @Override
+            public PathAddress convertToTargetAddress(PathAddress aliasAddress, AliasContext aliasContext) {
+                PathAddress target = this.getTargetAddress();
+                List<PathElement> result = new ArrayList<>(aliasAddress.size());
+                for (int i = 0; i < aliasAddress.size(); ++i) {
+                    PathElement element = aliasAddress.getElement(i);
+                    if (i == target.size() - 1) {
+                        final ModelNode operation = aliasContext.getOperation();
+                        final String stackName;
+
+                        if (ModelDescriptionConstants.ADD.equals(Operations.getName(operation)) && operation.hasDefined("type")) {
+                            stackName = operation.get("type").asString();
+                        } else {
+                            Resource root = null;
+                            try {
+                                root = aliasContext.readResourceFromRoot(PathAddress.pathAddress(result));
```

Comment:
This shouldn't block this PR, but passing the 'false' param to aliasContext.readResourceFromRoot should work fine. You still get the data on what the child resources are named, and you save the cost of cloning all the children.